### PR TITLE
Refactor NMS helm chart to match the layout with other subcharts.

### PR DESCRIPTION
**Pull request by reddydodda**
with this upgrade there is no impact to existing workload. helm upgrade wont destroy/re-create pods.

Signed-off-by: reddydodda <reddydodda@gmail.com>

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md [hunk 1]
```diff
@@ -403,36 +392,25 @@ Now that we've got an admin operator cert, we can deploy NMS. Edit the
...
     - name: orc8r-secrets-registry
 
   magmalte:
-    manifests:
-      secrets: true
-      deployment: true
-      service: true
-      rbac: false
-
+    create: true:
```
Comment (mpgermano) [inline docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md]: Extra colon?

File: docs/docusaurus/versioned_docs/version-1.0.0/orc8r/deploy_install.md [hunk 2]
```diff
@@ -297,36 +297,25 @@ metrics:
       worker-type: metrics
```
Comment (mpgermano) [inline docs/docusaurus/versioned_docs/version-1.0.0/orc8r/deploy_install.md]: We shouldn't update this doc as the changes won't exist for the version-1.0.0 tag

File: docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md [hunk 3]
```diff
@@ -297,36 +297,25 @@ metrics:
       worker-type: metrics
 
 nms:
+  enabled: false
```
Comment (mpgermano) [inline docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md]: Same for 1.0.1

File: orc8r/cloud/helm/orc8r/charts/nms/Chart.yaml [hunk 4]
```diff
@@ -8,7 +8,7 @@
 apiVersion: v1
 description: Magma NMS
 name: nms
-version: 0.1.2
+version: 0.1.5
```
Comment (mpgermano) [inline orc8r/cloud/helm/orc8r/charts/nms/Chart.yaml]: Rebase as this should be `0.1.6` now

**Review by mpgermano — CHANGES_REQUESTED** [review]
See comments. @xjtian Any other concerns here?

File: docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md (hunk 1 above)
Comment (reddydodda) [reply docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md]: fixed

File: docs/docusaurus/versioned_docs/version-1.0.0/orc8r/deploy_install.md (hunk 2 above)
Comment (reddydodda) [reply docs/docusaurus/versioned_docs/version-1.0.0/orc8r/deploy_install.md]: fixed

File: docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md (hunk 3 above)
Comment (reddydodda) [reply docs/docusaurus/versioned_docs/version-1.0.1/orc8r/deploy_install.md]: fixed

File: orc8r/cloud/helm/orc8r/charts/nms/Chart.yaml (hunk 4 above)
Comment (reddydodda) [reply orc8r/cloud/helm/orc8r/charts/nms/Chart.yaml]: fixed

**Pull request closed without merging**